src/screens/Settings/Resources.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Linking } from "react-native";
+import { useTranslation } from "react-i18next";
+import {
+  NanoSFoldedMedium,
+  LifeRingMedium,
+  NewsMedium,
+  FacebookMedium,
+  TwitterMedium,
+  GithubMedium,
+  ActivityMedium,
+} from "@ledgerhq/native-ui/assets/icons";
+import SettingsNavigationScrollView from "./SettingsNavigationScrollView";
+import SettingsCard from "../../components/SettingsCard";
+
+const Resources = () => {
```

Comment:
Why aren't these urls in the urls file? I understand you just copied this but, perhaps, it's a good time to put the urls there too 🤔

Reply:
Indeed, and not too difficult to do suppose, i will check this

Reply:
Done